lib/rules/parsing_method.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * @file ESLint rule to disallow unsanitized method calls to HTML parsing methods.
+ * @author Frederik Braun et al.
+ * @copyright 2015-2017 Mozilla Corporation. All rights reserved.
+ */
+"use strict";
+
+const createMethodsRule = require("../base_method");
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+const defaultRuleChecks = {
+    // check first parameter to `DOMParser.parseFromString()`
+    parseFromString: {
+        objectMatches: ["DOMParser"],
+        properties: [0],
+    },
+
+    // check first parameter to `Document.parseHTMLUnsafe()`
+    parseHTMLUnsafe: {
+        objectMatches: ["Document"],
```

Comment:
```suggestion
        objectMatches: ["document"],
```

Our matching is case-insensitive and use lowercase `document` elsewhere in the code.